e2e/fixtures/citizen/createCase/C100/c100ScreeningSections/permissionContent.ts

```diff
@@ -0,0 +1,16 @@
+export enum PermissionContent {
+  pageTitle = "Is there any reason that you would need permission from the court to make this application?",
+  body1 = "In some cases, the court will need to give permission before the application can be made.",
+  body2 = "You will not need permission if you are the:",
+  body3 = "If you are not sure if you need permission, see ",
+  list1 = "parent",
+  list2 = "guardian",
+  list3 = "special guardian",
+  aLink = "Section C of the Form CB1 guidance",
+  formLabel1 = "Yes",
```

Comment:
Just minor: Can we define the generic strings like 'Yes', 'No', 'Continue', 'Submit' etc. in a common/base file and read from there in our tests, we are re-defining them in each test class separately which can be avoided.

Reply:
Yes - I'll do that now